dsl/procedures/importDslFromGitNew/steps/deleteObjectsInChangeList.groovy

```diff
@@ -0,0 +1,113 @@
+/*
+  deleteObjectsInChangeList.groovy - Loop through a change list and perform delete actions on objects
+
+  Copyright 2022 Electric-Cloud Inc.
+
+  CHANGELOG
+  ----------------------------------------------------------------------------
+  2022-03-17  mayasse Initial Version
+
+*/
+import groovy.io.FileType
+import groovy.json.StringEscapeUtils
+import groovy.transform.BaseScript
+import com.electriccloud.client.groovy.ElectricFlow
+import com.electriccloud.client.groovy.apis.model.*
+import com.electriccloud.client.groovy.models.ActualParameter
+import java.io.File
+
+//noinspection GroovyUnusedAssignment
+//@BaseScript BaseObject baseScript
+
+$[/myProject/scripts/Utils]
+
+ElectricFlow ef = new ElectricFlow()
+
+// Is this an incremental import?
+def incremental = false
+try {
+    incremental = (ef.getProperty(propertyName:"/myJob/incrementalImport").property.value == "1") ? true : false
+} catch (Exception ex) {
+    //println "Could not get incrementaImport value due to: ${ex.message}"
+    incremental = false
+}
+
+println "Incremental Import: $incremental"
+
+def changeListText = "";
+if (incremental) {
+//    Gather change list text
+//    def fileName = "[dest]/change_list.json"
+//    // if file exists, is not a folder and is readable...
+//    def changeListFile = new File(fileName)
+//    if (changeListFile.exists() && changeListFile.isFile()) {
+//        changeListText = changeListFile.text
```

Comment:
do we need this commented out code here?

Reply:
We probably do not need the code.  There are 2 ways to retrieve the change list: into a property or into a file.  We chose "into a property" for importDslFromGitNew" to simplify the calling for users.